src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -41,6 +45,21 @@ public partial class UpdateDurableFunction
         logger = context.CreateReplaySafeLogger(EnsureArg.IsNotNull(logger, nameof(logger)));
 
         UpdateCheckpoint input = context.GetInput<UpdateCheckpoint>();
+        Uri uri = new Uri(input.Endpoint, UriKind.Absolute);
+        var callerClaims = new List<KeyValuePair<string, string>>
+        {
+            new ("operation_id", context.GetOperationId().ToString()),
```

Comment:
Should this key be defined somewhere?

Reply:
We are not using this key anywhere else. So it is safe to hardcode for now.